### ISSUE TITLE
Bug: "GET /clan" doesn't return language enum, ageRange enum or goal enum.

### ISSUE THREAD

**Issue by BillTheBeast** [Alt-Org/Altzone-Server#333]
## What doesn't work?

"GET /clan" doesn't return language enum, ageRange enum or goal enum.

## What is the endpoint and method? (if applicable)

Endpoint: /clan

Method: GET

## How can it be reproduced?

1. Call Get /clan

## What should it do instead?

Return language, ageRange and goal enums together with the rest of the data.

## Anything else you want to add?

Swagger does say that it should return these, but I just checked and it actually doesn't.

**Comment by MikhailDeriabin** [comment]
@BillTheBeast 

What version of API are you using?

It does return these properties from _dev_ and _latest-release_ versions:

- https://devapi.altzone.fi/clan
- https://devapi.altzone.fi/latest-release/clan

**Comment by BillTheBeast** [comment]
Yeah, it seems that for some reason (I made a new field for the devaddress and forgot to change this call) this pointed to the wrong address. Also I forgot that the "normal" address is very behind. Part of the reason I was confused was because the call did include labels, which were added around th same time as these.

Anyway, yeah it works now.
